breathecode/feedback/tests/urls/tests_academy_survey.py

```diff
@@ -0,0 +1,256 @@
+            },
+            'avg_score': model['survey'].avg_score,
+            'response_rate': model['survey'].response_rate,
+            'status': model['survey'].status,
+            'duration': '86400.0',
+            'created_at': self.bc.datetime.to_iso_string(model['survey'].created_at),
+            'sent_at': None,
+            'public_url': 'https://nps.breatheco.de/survey/1'
+        }]
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    @patch('breathecode.feedback.signals.survey_answered.send', MagicMock())
+    def test_academy_survey__get__with_status_answered(self):
+        """Test /academy/survey with status answered"""
+
+        from breathecode.feedback.signals import survey_answered
+
+        self.headers(academy=1)
+        survey = {'status': 'SENT'}
+        answer = {'status': 'ANSWERED'}
+        model = self.bc.database.create(
+            authenticate=True,
+            academy=True,
+            profile_academy=True,
+            role='read_survey',
+            survey=survey,
+            answer=answer,
+            capability='read_survey',
+        )
+
+        db = self.model_to_dict(model, 'survey')
+        url = reverse_lazy('feedback:academy_survey')
+        response = self.client.get(url)
+        json = response.json()
+        expected = [{
+            'id': model['survey'].id,
+            'lang': model['survey'].lang,
+            'cohort': {
+                'id': model['cohort'].id,
+                'slug': model['cohort'].slug,
+                'name': model['cohort'].name
+            },
+            'avg_score': model['survey'].avg_score,
+            'response_rate': model['survey'].response_rate,
+            'status': model['survey'].status,
+            'duration': '86400.0',
+            'created_at': self.bc.datetime.to_iso_string(model['survey'].created_at),
+            'sent_at': None,
+            'public_url': 'https://nps.breatheco.de/survey/1'
+        }]
+        expected_args_list = [call(instance=model.answer, sender=model.answer.__class__)]
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(survey_answered.send.call_args_list, expected_args_list)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    @patch('breathecode.feedback.signals.survey_answered.send', MagicMock())
+    @patch('breathecode.feedback.tasks.process_answer_received.delay', MagicMock())
+    def test_academy_survey__get__with_status_answered_call_signal(self):
```

Comment:
`test_academy_survey__get__with_status_answered` is similar to this test, remove this test